src/indexer.c

```diff
@@ -231,6 +232,11 @@ static void writeCurEntries(DocumentIndexer *indexer, RSAddDocumentCtx *aCtx, Re
         invidx->fieldMask |= entry->fieldMask;
       }
     }
+    
+    if (spec->suffixMask & entry->fieldMask && entry->term[0] != '+') {
```

Comment:
Is this to avoid indexing the expender terms? If so IIRC there are more chars we want to avoid...

Reply:
I do not see if this comment was fixed, was it?